### PR TITLE
Add example for `Smooth*` graphics instructions

### PR DESCRIPTION
**Pull request by mak8kammerer**
PR #8309 added a new graphics section with antialiasing ( @DexerBR, thank you). But there are no examples. I created one small demo script.

image

Maintainer merge checklist
* [x] Title is descriptive/clear for inclusion in release notes.
* [x] Applied a `Component: xxx` label.
* [ ] Applied the `api-deprecation` or `api-break` label.
* [ ] Applied the `release-highlight` label to be highlighted in release notes.
* [x] Added to the milestone version it was merged into.
* [ ] **Unittests** are included in PR.
* [ ] Properly documented, including `versionadded`, `versionchanged` as needed.

### CODE REVIEW

**Comment by misl6** [comment]
Seen this after a long time and feel guilty to ask, but, you can please add a section on top of the file which explains this example?

E.g. : https://github.com/kivy/kivy/blob/dcd8fb2a6ae3f96789d51e00e1ff2f3c2fc339ef/examples/canvas/canvas_stress.py#L1-L10

File: examples/canvas/antialiasing.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
+'''
+Antialisiang Example
```
Comment (misl6) [inline examples/canvas/antialiasing.py:2]:
```suggestion
Antialiasing Example
```

File: examples/canvas/antialiasing.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+====================
+
+Kivy 2.3.0 introduced several vertex instrustions with with antialisiang:
```
Comment (misl6) [inline examples/canvas/antialiasing.py:5]:
```suggestion
Kivy 2.3.0 introduced several vertex instructions with with antialiasing:
```

Comment (mak8kammerer) [reply examples/canvas/antialiasing.py]: My bad, sorry :(

File: examples/canvas/antialiasing.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
+'''
+Antialiasing Example
+====================
+
+Kivy 2.3.0 introduced several vertex instructions with with antialiasing:
```
Comment (misl6) [inline examples/canvas/antialiasing.py:5]:
```suggestion
Kivy 2.3.0 introduced several vertex instructions with antialiasing:
```

**Review by misl6 — COMMENTED** [review]
Another small typo

File: examples/canvas/antialiasing.py (hunk 3 above)
Comment (mak8kammerer) [reply examples/canvas/antialiasing.py]: Thank You!!!

**Review by misl6 — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**